low-level/files/opt/seagate/sspl/bin/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -325,11 +329,28 @@ def sspl_bundle_cleanup():
     if os.path.exists(sspl_bundle_tmp):
         shutil.rmtree(sspl_bundle_tmp)
 
+def validate_sb_request():
+    """Validating upcomming sb request."""
+    valid_request = True
+    if os.path.isfile(onGoingRequestPath):
+        last_reboot = int(psutil.boot_time())
+        req_start_time = int(pathlib.Path(onGoingRequestPath).stat().st_mtime)
+        current_time = int(time.time())
+        # Check if system is rebooted in between or request is expired.
+        # If Yes, Then clear existing request cache.
+        is_req_expired = (current_time - req_start_time) > SB_REQUEST_TIMEOUT
+        if last_reboot > req_start_time or is_req_expired:
+            sspl_bundle_cleanup()
+        else:
```

Comment:
I dont see any test code added. 

test case: 
for i in 0... 100; do
    Start Support Bundle 
    [ $? -ne 0 ] && echo "test failed" && break
    sleep 1
    killall -9 support_bundle process name
done

Reply:
Done